ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc

```diff
@@ -238,6 +276,19 @@ bool snpe_subplugin::parse_custom_prop (const char *custom_prop)
           nns_loge ("Unknown cpu_fallback option");
           invalid_option = true;
         }
+      } else if (g_ascii_strcasecmp (option[0], "OutputLayer") == 0) {
+        gchar **names = g_strsplit (option[1], ";", -1);
+        for (guint i = 0; i < g_strv_length (names); ++i) {
```

Comment:
```suggestion
        guint len_str_arr = g_strv_length (names)
        for (guint i = 0; i <len_str_arr; ++i) {
```

It is better to use a local variable here.

Reply:
I followed your suggestion. Thanks!